mobile-widgets/qml/icons/sigma.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.0//EN"
+"http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<!-- Created with Sodipodi ("http://www.sodipodi.com/") -->

```

Comment:
so we borrowed a lot of our icons from Breeze and from the Google Material Design icon set.
Did you go straight for your own design or did you look if one of those two sources have something reasonable to use?
I'm fine with a custom icon (haven't looked at this one, I'm on a plane), but I wonder if we should try to find something by the same design team, first, before doing our own.
Again, apologies for not looking at it before commenting - maybe this is perfect :-)

Reply:
I did not find the sigma symbol, but maybe they named it differently.

and no it does not depend on (3) solely (1) and (2).